src/components/InlineSpinner.vue

```diff
@@ -14,18 +14,22 @@
 </template>
 
 <script>
-export default {
-  name: 'InlineSpinner',
+import { defineComponent } from 'vue'
+
+export default defineComponent({
   props: {
     size: {
       default: 32,
       type: Number
     }
   },
-  data () {
+  setup (props) {
+    const spinnerSize = props.size
```

Comment:
```suggestion
    const { size: spinnerSize } = toRefs(props)
```

Use `toRefs` when destructing props, otherwise the reactivity will be lost.